hack/demo/helm_install_crossplane_master.sh

```diff
@@ -12,43 +11,22 @@ kubectl create namespace crossplane-system
 version=$(helm search repo crossplane --devel | awk '$1 == "crossplane-master/crossplane" {print $2}')
 helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version $version --devel
```

Comment:
We install crossplane here and install oam-k8s-runtime below, will it conflict? As far as I know, crossplane package still containe oam-k8s-runtime, right?

Reply:
> crossplane package still containe oam-k8s-runtime, right?

No. It does not now